lib/plugins/inspec-parallel/lib/inspec-parallel/exec_command.rb

```diff
@@ -83,8 +83,13 @@ def dry_run_commands
 
       def read_options_file
         opts = []
+        content = []
         begin
-          content = content_from_file(cli_options_to_parallel_cmd[:option_file])
+          if cli_options_to_parallel_cmd[:option_file] == "-"
+            content = $stdin.read&.split("\n") || []
```

Comment:
Are we obliged to slurp the entire STDIN at once? That will limit things that want to pipe chains of things. Is there any way we can read one line at a time?

Reply:
Sure let me look into that.

Reply:
So you are suggesting a scenario where inspec parallel list is called multiple times and then the output for each pipe should get read properly. Correct me if I am wrong.